djangocms_versioning_filer/monkeypatch/admin/fileadmin.py

```diff
@@ -12,10 +12,7 @@ def inner(self):
         current_filename = self.instance.file.name.split('/')[-1]
         cleaned_data = func(self)
         file = cleaned_data.get('file')
-        if (
-            file and
-            file.name.split('/')[-1] != current_filename
-        ):
+        if file and file.name.split('/')[-1].lower() != current_filename.lower():
```

Comment:
default upload_to function is doing slugify on name and it's changing it to lowercase. As I remember Mac/Windows are case insensitive with file naming.

Reply:
Ofc someone can provide his own upload_to function, that's totally changing a filename. Idk what to do with that situation.